### ISSUE TITLE
animated stickers size

### ISSUE THREAD

**Issue by m-koushan** [NekoX-Dev/NekoX#783]
**Describe the bug**
animated stickers are eating my storage

**To Reproduce**
1.Go to settings-data and storage-storage usage
2.Click on device storage
3.for me always theres a new section that name is animated stickers, this section is always consuming so much storage and idk why
Here is an image from mine (i cleared all of my data and after that i checked less than 20 seconds)

**Logcat**

**Expected behavior**
i expected that the animated stickers storage is for stickers that are animated but i cant understand how its possible consuming this much storage even i dont have any animated stickers too

**Screenshots**
![Screenshot_2022-03-15-18-31-16-961_nekox.messenger_1.jpg](https://user-images.githubusercontent.com/50526840/158408447-35de33c8-c347-4c0b-9a74-58c97b5d58ff.jpg)

**Smartphone (please complete the following information):**
 - Device: Redmi Note 8 Pro
 - OS: Android 11
 - Version 8.6.1

**Additional context**

**Comment by Attorelle** [comment]
not a bug

**Comment by jpzex** [comment]
As you may know, the app needs some space for it to function, the storage cannot be completely wiped. It may apply to the stickers section too. Maybe this occupied storage is the responsible to store something that runs the animation or stores the stickers other people send to you on groups or private messages.

**Comment by m-koushan** [comment]
its been a while i was using the app and without downloading anything that accounts as animated stickers and you see here its using 1.1GB of my storage, in what world a messaging app needs this much storage to just handle something that i didnt even see or downloaded?!
![Screenshot_2022-04-07-18-43-01-699_nekox messenger_1](https://user-images.githubusercontent.com/50526840/162220657-3af6dfef-ec8c-4f94-b774-e10b874953db.jpg)

I was using the orginal telegram too and there is the same problem there

**Comment by m-koushan** [comment]
Hi I changed the Storage Path in the settings to run some tests about reactions stuff.
I discovered that telegram and nekoxgram in my phone everytime someone or me react an emoji on a message, it will claim some storage space. everything is normal now. but in my phone it actually claims storage for every message that has its first reaction emoji.
for example what it should be is if someone react with like emoji for first time in whole chat, the app should claim some space around 4mb. and after that if anyone react with like emoji on other messages, it should use that cached file which created by app for the first time.
but in my phone the app everytime creates a new file for it. and thats why i had storage issue with these things.
anyone can help what should i do? and i only saw this on my phone(Redmi note 8 pro)

also the files for the reaction emojis are stored in 
``[your nekox data folder]/files/caches/acache``
